package.json

```diff
@@ -21,7 +21,7 @@
     "@babel/preset-react": "^7.10.4",
     "@oacore/eslint-config-react": "^1.0.6",
     "@oacore/prettier-config": "^1.0.6",
-    "@oacore/stylelint-config-base": "^1.0.6",
+    "@oacore/stylelint-config-base": "^2.0.0-stylelint",
```

Comment:
You should have kept just 2.0.0 without the suffix.

Reply:
What's the problem with the prerelease tag?

Reply:
Nothing is wrong but now all @oaocore/configs packages will need to be upgraded to this version with prerelease tag and it doesn't look good to my eye 😄

Reply:
Don't update then 😄